vital/types/metadata_traits.cxx

```diff
@@ -69,18 +70,34 @@ ::metadata_traits()
   KWIVER_VITAL_METADATA_TAGS( TABLE_ENTRY )
 
 #undef TABLE_ENTRY
+
+  // Create name table
+#define NAME_TABLE_ENTRY(TAG, NAME, TYPE, ...) \
```

Comment:
BTW, I realize trying to uncrustify this is probably a hot mess and not something we want to deal with right this instant, but it would be nice to try to follow style, or at least *be consistent*, with the new code. (Spaces inside parentheses, brackets, etc.)